components/x-article-save-button/src/ArticleSaveButton.scss

```diff
@@ -0,0 +1,36 @@
+@import 'o-icons/main';
+@import 'o-colors/main';
+@import 'o-typography/main';
+
+$icon-size: 38px;
+
+@mixin background-icon($name, $color) {
+	@include oIconsGetIcon($name, oColorsGetPaletteColor($color), $icon-size, $iconset-version: 1);
+}
+
+.article-save-button {
+	display: inline-block;
+}
+
+.article-save-button__button {
+	@include background-icon(bookmark-outline, #000000);
+	@include oTypographySans($scale: -2);
+	border: 0;
+	padding: ($icon-size - 5px) 0 0 0;
+	width: $icon-size;
+	color: getColor('black');
+
+	&:focus {
+		outline: 3px solid getColor('teal-40');
+		outline-offset: 3px;
+	}
+
+	&:hover,
+	&:focus {
+		@include background-icon(bookmark, oColorsGetPaletteColor('black-50'));
+	}
+
+	.article-save-button--saved &:not(:hover):not(:focus) {
```

Comment:
If this class name is moved onto the button element instead of the container then the selector wouldn't need to negate all of the pseudo-classes which would be neater IMO

Reply:
Yep, sure. I was just allowing for other changes in styling for the saved state - stuff outside of the `button`. There's no immediate use case for that though. Actually I think a separate `--saved` class is not necessary, and the `aria-pressed` attribute could be used instead.